nixos/modules/programs/nano.nix

```diff
@@ -12,16 +12,16 @@ in
 
       nanorc = lib.mkOption {
         type = lib.types.lines;
-        default = "";
+        default = ''
+          set nowrap
+          set tabstospaces
+          set tabsize 2
+          include "${pkgs.nano}/share/nano/*.nanorc"
```

Comment:
It's not necessary to set `defaultText` now; it's a hack to avoid incurring lots of work to render the manual (typically whenever the default value refers to a package or other non-trivial artefact).